### PR TITLE
feat: profile entry point in wallet dropdown

### PR DESCRIPTION
**Pull request by cbachmeier**
- refactor sell and select page to profile page
- adjust url pathing from `/nfts/sell` to `/profile`
- Remove 'Sell NFTs' entry point
- Add 'View and sell NFTs' entry point in Wallet Dropdown
- Add Header to Profile page with account details
  - Handles ENS and Avatar
- Refactor AvatarIcon to take an optional `size` parameter

### CODE REVIEW

File: src/nft/components/profile/view/ProfileAccountDetails.tsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  const [isCopied, setCopied] = useCopyClipboard()
+  const copy = useCallback(() => {
+    setCopied(account || '')
+  }, [account, setCopied])
+
+  return (
+    <Row className={headlineLarge} marginBottom="48" gap="12">
+      <Identicon size={44} />
+      <Box textOverflow="ellipsis" overflow="hidden">
+        {ENSName || shortenAddress(account ?? '')}
```
Comment (JackShort) [inline src/nft/components/profile/view/ProfileAccountDetails.tsx:24]: honestly this component should just return `null` if account is undefined

Comment (JackShort) [reply src/nft/components/profile/view/ProfileAccountDetails.tsx:24]: also `??` is better than `||` when comparing nullish values

File: src/nft/components/profile/view/ProfileAccountDetails.tsx [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+      <MouseoverTooltip
+        text={
+          <Box className={caption} color="textPrimary">
+            {isCopied ? <Trans>Copied!</Trans> : <Trans>Copy</Trans>}
+          </Box>
+        }
+        placement="right"
+      >
+        <Box paddingX="12" borderRadius="12" cursor="pointer" className={lightGrayOverlayOnHover} onClick={copy}>
+          <Copy strokeWidth={1.5} size={20} />{' '}
```
Comment (JackShort) [inline src/nft/components/profile/view/ProfileAccountDetails.tsx:35]: is this necessary `{' '}`?

File: src/nft/components/profile/view/ProfileAccountDetails.tsx [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+import { caption, headlineLarge, lightGrayOverlayOnHover } from 'nft/css/common.css'
+import { useCallback } from 'react'
+import { Copy } from 'react-feather'
+import { shortenAddress } from 'utils'
+
+export const ProfileAccountDetails = () => {
+  const { account, ENSName } = useWeb3React()
+  const [isCopied, setCopied] = useCopyClipboard()
+  const copy = useCallback(() => {
+    setCopied(account || '')
```
Comment (JackShort) [inline src/nft/components/profile/view/ProfileAccountDetails.tsx:17]: `??` is better than `||` for nullish values

File: src/nft/components/profile/view/ProfilePage.tsx [hunk 4]
```diff
@@ -245,67 +246,69 @@ export const SelectPage = () => {
...
+      width="full"
+      paddingLeft={{ sm: '16', md: '52' }}
+      paddingRight={{ sm: '0', md: '72' }}
+      paddingTop={{ sm: '16', md: '40' }}
+    >
+      <Row alignItems="flex-start" position="relative">
         <FilterSidebar SortDropdown={SortWalletAssetsDropdown} />
 
         {(!isMobile || !isFiltersExpanded) && (
           // @ts-ignore
```
Comment (JackShort) [inline src/nft/components/profile/view/ProfilePage.tsx:259]: is this necessary if we are removing `AnimatedBox`?

File: src/nft/components/profile/view/ProfileAccountDetails.tsx (hunk 2 above)
Comment (cbachmeier) [reply src/nft/components/profile/view/ProfileAccountDetails.tsx:35]: This is auto-added by the ts linter, you can find it throughout the app

File: src/nft/components/profile/view/ProfileAccountDetails.tsx (hunk 1 above)
Comment (cbachmeier) [reply src/nft/components/profile/view/ProfileAccountDetails.tsx:24]: We have a higher level check on sell.tsx to not show this page when account is null, this check was more to resolve compile warnings

File: src/nft/components/profile/view/ProfilePage.tsx (hunk 4 above)
Comment (cbachmeier) [reply src/nft/components/profile/view/ProfilePage.tsx:259]: Can be removed thanks to the newly added `any` cast of AnimatedBox. That being said its deprecation is a long term goal as part of the greater sprinkles deprecation.

**Review by JackShort — APPROVED** [review]
lgtm

**Pull request merged**